Meden-Agan-27/travel-worms
# Navbar fixes

What I did:
removed the dropdown within the dropdown in mobile view

How to test:
check there are no longer 2 dropdowns and check that all the functions work.
You can check if it works in browser now too, it did for me, but the focus is on mobile.
